app/reducers/gameState.test.js

```diff
@@ -213,26 +213,26 @@ describe('END_TURN', () => {
       ).toMatchObject({ prestige: 3 }));
 
     describe('replenishes hand', () => {
-      test('draw five', () =>
+      test('draw a full hand', () =>
```

Comment:
The handsize change killed these tests; just fixed them in passing.

Reply:
Sorry about that!  Nice catch.